test_schemas/en/test_custom_page_titles.json

```
@@ -841,6 +841,37 @@
                     "text": "{person_name}"
                 }
             }
+        },
+        {
+            "id": "question-section",
+            "groups": [
+                {
+                    "blocks": [
+                        {
+                            "type": "Question",
+                            "id": "name-block",
+                            "page_title": "Custom question page title",
+                            "question": {
+                                "answers": [
+                                    {
+                                        "id": "name-answer",
+                                        "label": "What is your name?",
+                                        "max_length": 20,
+                                        "mandatory": false,
+                                        "q_code": "0",
+                                        "type": "TextField"
+                                    }
+                                ],
+                                "id": "name-question",
+                                "title": "Title",
+                                "type": "General"
+                            }
```

q_code is not needed and the question text should be in the question title.
```suggestion
                            "question": {
                                "answers": [
                                    {
                                        "id": "name-answer",
                                        "label": "Name",
                                        "max_length": 20,
                                        "mandatory": false,
                                        "type": "TextField"
                                    }
                                ],
                                "id": "name-question",
                                "title": "What is your name?",
                                "type": "General"
                            }
```